keyboards/ymdk/ymd40/v2/v2.h

```diff
@@ -29,3 +29,51 @@
     { K20,   K21,   K22,   K23,   K24,   K25,   K26,   K27,   K28,   K29,   K2A,   K2B   }, \
     { K30,   K31,   K32,   K33,   K34,   K35,   K36,   K37,   K38,   K39,   K3A,   K3B   }, \
 }
+
+#define LAYOUT_ymd40v2_2x2u( \
```

Comment:
These should be named `LAYOUT_ortho_4x12_*` to match the existing layout.

Reply:
Hi Ryan,

I was just inferring from using the planck as a baseline.
LAYOUT_planck_2x2u
LAYOUT_ortho_4x12

Should I proceed with changing the names as per your suggestion?

Reply:
Yeah, those Planck layout names are not ideal, we prefer using more generic names so they can be reused for any other keyboard with the same layout - see the Layouts documentation: https://docs.qmk.fm/#/feature_layouts

Reply:
Thanks for the advice.